### PR TITLE
tidb-in-kubernetes: add offline pump document

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Add offline pump document

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1757

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

File: dev/tidb-in-kubernetes/maintain/backup-and-restore.md [hunk 1]
```diff
@@ -126,3 +126,35 @@ The `pingcap/tidb-backup` helm chart helps restore a TiDB cluster using backup d
 Incremental backup uses [TiDB Binlog](/dev/reference/tidb-binlog-overview.md) to collect binlog data from TiDB and provide real-time backup and replication to downstream platforms.
 
 For the detailed guide of maintaining TiDB Binlog in Kubernetes, refer to [TiDB Binlog](/dev/tidb-in-kubernetes/maintain/tidb-binlog.md).
+
+### Decrease the capability of Pump
```
Comment (weekface) [inline dev/tidb-in-kubernetes/maintain/backup-and-restore.md]: Use `Scale down` instead of `Decrease the capability of`?

**Comment by juliezhang1112** [comment]
PTAL @yikeke

File: v3.0/tidb-in-kubernetes/maintain/backup-and-restore.md [hunk 2]
```diff
@@ -128,7 +128,7 @@ Incremental backup uses [TiDB Binlog](/v3.0/reference/tidb-binlog-overview.md) t
 
 For the detailed guide of maintaining TiDB Binlog in Kubernetes, refer to [TiDB Binlog](/v3.0/tidb-in-kubernetes/maintain/tidb-binlog.md).
 
-### Decrease the capability of Pump
+### Scale Pump in
 
 To decrease the capability of Pump, make Pump nodes offline and then run the `helm upgrade` command to delete Pump pods.
```
Comment (weekface) [inline v3.0/tidb-in-kubernetes/maintain/backup-and-restore.md]: `To decrease the capability of Pump`

this should be modified

File: dev/tidb-in-kubernetes/maintain/backup-and-restore.md [hunk 3]
```diff
@@ -126,3 +126,35 @@ The `pingcap/tidb-backup` helm chart helps restore a TiDB cluster using backup d
 Incremental backup uses [TiDB Binlog](/dev/reference/tidb-binlog-overview.md) to collect binlog data from TiDB and provide real-time backup and replication to downstream platforms.
 
 For the detailed guide of maintaining TiDB Binlog in Kubernetes, refer to [TiDB Binlog](/dev/tidb-in-kubernetes/maintain/tidb-binlog.md).
+
+### Scale Pump in
+
+To decrease the capability of Pump, make Pump nodes offline and then run the `helm upgrade` command to delete Pump pods.
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/maintain/backup-and-restore.md]:
```suggestion
To scale Pump in, you decrease the number of Pump nodes by taking the nodes offline one at a time and running the `helm upgrade` command to delete the corresponding Pump Pod. The steps are as follows:
```
"Pod" needs to be capitalized. ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/

File: v3.0/tidb-in-kubernetes/maintain/backup-and-restore.md (hunk 2 above)
Comment (juliezhang1112) [reply v3.0/tidb-in-kubernetes/maintain/backup-and-restore.md]: Got it.

File: dev/tidb-in-kubernetes/maintain/backup-and-restore.md [hunk 4]
```diff
@@ -126,3 +126,35 @@ The `pingcap/tidb-backup` helm chart helps restore a TiDB cluster using backup d
...
+
+1. Make Pump nodes offline from the TiDB cluster
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/maintain/backup-and-restore.md]:
```suggestion
1. Make the Pump node offline from the TiDB cluster
```

File: dev/tidb-in-kubernetes/maintain/backup-and-restore.md [hunk 5]
```diff
@@ -126,3 +126,35 @@ The `pingcap/tidb-backup` helm chart helps restore a TiDB cluster using backup d
...
+
+    Suppose there are 3 Pump nodes, and you want to get the third node offline and modify `<ordinal-id>` to `2`, run the following command. `<version>` is the current version of TiDB.
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/maintain/backup-and-restore.md]:
```suggestion
    Suppose there are 3 Pump nodes, and you want to get the third node offline and modify `<ordinal-id>` to `2`, run the following command (`<version>` is the current version of TiDB).
```

File: dev/tidb-in-kubernetes/maintain/backup-and-restore.md [hunk 6]
```diff
@@ -126,3 +126,35 @@ The `pingcap/tidb-backup` helm chart helps restore a TiDB cluster using backup d
...
+
+    Then, check the log output of Pump. If Pump outputs `pump offline, please delete my pod`, the state of the Pump node is successfully switched to `offline`.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl logs -f -n <namespace> <release-name>-pump-<ordinal-id>
+    ```
+
+2. Delete Pump pods
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/maintain/backup-and-restore.md]:
```suggestion
2. Delete the corresponding Pump Pod
```

File: dev/tidb-in-kubernetes/maintain/backup-and-restore.md [hunk 7]
```diff
@@ -126,3 +126,35 @@ The `pingcap/tidb-backup` helm chart helps restore a TiDB cluster using backup d
...
+
+    Modify `binlog.pump.replicas` in the `values.yaml` file to `2` and then, run the following command to delete Pump pods.
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/maintain/backup-and-restore.md]:
```suggestion
    Modify `binlog.pump.replicas` in the `values.yaml` file to `2` and then run the following command to delete the Pump Pod.
```

**Pull request merged**